draft-irtf-cfrg-hpke.md

```diff
@@ -489,10 +504,12 @@ context. The key schedule inputs are as follows:
 * `mode` - A one-byte value indicating the HPKE mode, defined in {{hpke}}.
 * `zz` - A KEM shared secret generated for this transaction
 * `info` - Application-supplied information (optional; default value
-  "")
+  "") of maximum length 65535 bytes.
 * `psk` - A pre-shared secret held by both the sender
-  and the recipient (optional; default value `zero(Nh)`).
+  and the recipient (optional; default value `zero(Nh)`) of maximum
+  length 255 bytes.
```

Comment:
Also don't need this limit anymore.

Reply:
This one I pulled out of thin air. I can make it larger to match. :)